capstone/src/main/java/com/google/step/clients/RestaurantClient.java

```diff
@@ -0,0 +1,116 @@
+package com.google.step.clients;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.GeoPt;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.step.data.Restaurant;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.NoSuchElementException;
+
+public class RestaurantClient {
+  private static DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+
+  /**
+   * Puts a single entity of any kind into datastore.
+   * @param restaurant, to be processed into an Entity and put into datastore
+   * @param email, separate from Restaurant class and passed from UserService in the servlet
+   */
+  public void putRestaurant(Restaurant restaurant, String email) {
+    // Retrieve restaurant properties
+    String name = restaurant.getName();
+    GeoPt location = restaurant.getLocation();
+    String story = restaurant.getStory();
+    List<String> cuisineList = restaurant.getCuisine();
+    String phone = restaurant.getPhone();
+    String website = restaurant.getWebsite();
+    String status = restaurant.getStatus();
+
+    // Parse into a RestaurantInfo Entity
+    Entity restaurantInfo = new Entity("RestaurantInfo");
```

Comment:
Does this mean we don't support updates? That's okay to start; no need to bother with the UI for it until the MVP is done and you have time.

Reply:
We hadn't written anything to support updates yet but hopefully with the new restaurantKey structure we can integrate it once the MVP is done!